code/src/terrat_ui_js_service/terrat_ui_js_service_comp_billing.ml

```diff
@@ -0,0 +1,36 @@
+module At = Brtl_js2.Brr.At
+
+module Make (Vcs : Terrat_ui_js_service_vcs.S) = struct
+  module State = Terrat_ui_js_service_state.Make (Vcs)
+
+  let stripe_portal = "https://billing.stripe.com/p/login/00geXngL2cQR0YofYY"
+
+  let billing_info state =
+    let app_state = Brtl_js2.State.app_state state in
+    let { State.selected_installation; _ } = app_state.State.v in
+    let features = Vcs.Installation.tier_features selected_installation in
+    let nupm =
+      match features.Terrat_ui_js_service_vcs.Tier.num_users_per_month with
+      | Some i -> CCInt.to_string i
+      | None -> "Unlimited users"
+    in
+    let trial_ends_at =
+      match Vcs.Installation.trial_ends_at selected_installation with
```

Comment:
This can become `CCOption.map_or ~default:"No expiration" Brtl_js2_datatime.to_iso_string @@ Vcs.Installation.trial_ends_at selected_installation`

Reply:
into it pt trois!